src/FSAL/FSAL_CORTXFS/mds.c

```diff
@@ -1137,8 +1137,9 @@ kvsfs_layoutcommit(struct fsal_obj_handle *obj_hdl,
 		container_of(obj_hdl,
 			     struct kvsfs_fsal_obj_handle,
 			     obj_handle);
-	// kvsfs_handle = myself->handle;
-	gtbl_layout_rmv_elem(myself->handle);
+	const struct kvsfs_file_handle * kvsfs_handle
+		= (const struct kvsfs_file_handle *)myself->handle;
```

Comment:
This needs few more steps:
1. extract myself->handle which is of type struct cfs_fh
2. convert struct cfs_fh to struct kvsfs_file_handle which is needed by gtbl_layout_rmv_elem
3. pass the converted struct kvsfs_file_handle to gtbl_layout_rmv_elem

Reply:
Regarding point 2, below are the definitions of struct cfs_fh and struct kvsfs_file_handle:
	struct cfs_fh {
			struct kvnode f_node;
			struct cfs_fs *fs;
			struct cfs_fh_key key;
	};
	struct kvsfs_file_handle {
	        uint64_t fs_id;
	        cfs_ino_t kvsfs_handle;
	};
As you can see, it looks the conversion is not possible because the structures are different. Do you know of or suggest any other approach for this?